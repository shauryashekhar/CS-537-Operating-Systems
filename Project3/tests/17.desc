& at the end. Adding background jobs to populate jobs table.