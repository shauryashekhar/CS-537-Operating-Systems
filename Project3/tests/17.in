sleep 2 &
jobs
exit
