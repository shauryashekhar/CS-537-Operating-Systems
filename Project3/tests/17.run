./mysh tests/17.in
