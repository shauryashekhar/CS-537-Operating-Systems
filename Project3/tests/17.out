sleep 2 &
jobs
0 : sleep 2
exit
